tests/pytest/oauth/test_views.py

```diff
@@ -30,6 +32,17 @@ def test_login(mocker, app_request):
     assert not session.logged_in(app_request)
 
 
+def test_login_analytics(mocker, app_request):
+    spy = mocker.patch("benefits.oauth.analytics.started_sign_in")
+
+    mock_client = mocker.patch.object(benefits.oauth.views, "oauth_client")
+    mock_client.authorize_redirect.return_value = HttpResponse("authorize redirect")
+
+    login(app_request)
+
+    spy.assert_called_once()
+
+
 @pytest.mark.request_path("/oauth/login")
```

Comment:
This confused me more: should this be `"oauth/authorize"` instead of `"oauth/login"`?

Reply:
Yep, good catch.

```suggestion
@pytest.mark.request_path("/oauth/authorize")
```

Reply:
Yeah I think the better answer here is to probably just delete these, since it is more confusing than necessary

Reply:
(you and @afeld are correct, but the value isn't actually used...)

Reply:
Thanks for the confirmation @afeld !

If the value isn't actually used, then I would also vote to remove them.

Reply:
Removed them in [fd3ddc5](https://github.com/cal-itp/benefits/pull/452/commits/fd3ddc529ffe067b4c4ca7ddbb2261b09b4bf239)